Source/objects.cpp

```diff
@@ -5168,7 +5168,7 @@ void SyncOpObject(int pnum, int cmd, int i)
 	case OBJ_LEVER:
 	case OBJ_L5LEVER:
 	case OBJ_SWITCHSKL:
-		OperateLever(pnum, i);
+		OperateLever(i, pnum == MyPlayerId);

```

Comment:
nit: Could we introduce a löocal `sendmsg` variable (like in `OperateObject`)?
Then we would have only one comparison with `pnum` in `SyncOpObject` and I would expect we could need this more often in the future. 😉 
Note: Didn't use approve, else the pr gets automerged and I'm unsure if you want to change this or not. 🙂

Reply:
I was thinking the number of comparisons here was exactly low enough that I didn't want to add it yet, but if I find one more later then I will add it :)